Source/WebKit/WebKitSwift/MarketplaceKit/WKMarketplaceKit.swift

```diff
@@ -21,32 +21,27 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
 // THE POSSIBILITY OF SUCH DAMAGE.
 
-#if os(iOS) && !os(visionOS) && canImport(MarketplaceKit)
+#if USE_APPLE_INTERNAL_SDK && HAVE_MARKETPLACE_KIT
```

Comment:
Pushing USE_APPLE_INTERNAL_SDK checks down into the code makes sense, though I wonder if it _should_ be this way. MarketplaceKit is a public framework; maybe we can make this work on the open source build without much effort?

Reply:
yes, that was my plan for my next PR! just wanted to not make too many different changes in a single PR